libfs/fs.go

```diff
@@ -79,16 +88,20 @@ func NewFS(ctx context.Context, config libkbfs.Config,
 	log.CDebugf(ctx, "Made new FS for TLF=%s, subdir=%s",
 		tlfHandle.GetCanonicalName(), subdir)
 
+	// Use the canonical unix path for default locking namespace, as this needs
+	// to be the same across all platforms.
+	unixFullPath := path.Join("/keybase", tlfHandle.Type().String(), subdir)
```

Comment:
I know I've mentioned this in the past with one of Strib's PRs, but just make sure that `subdir` is slashed correctly on Windows vs Unix.

Reply:
This is intentionally forward slashed since it has to be consistent across platforms. It's just used as part of lock ID hashing source, but never for locating files on the filesystem.